landing_page/static/landing_page/css/index.css

```diff
@@ -38,15 +38,15 @@ footer {
 
 .tooltiptext {
   display: none;
-  width: 200px;
-  height: auto;
-  background-color: #00000095;
+  width: 234px;
+  height: 350px;
+  background-color: #00000099;
```

Comment:
I think this should be a bit darker to make the text more legible.

Reply:
that's the darkest it can be... pitch black with the minimum amount of transparency

Reply:
i see... ;-; 
 Maybe it could be a solid colored background similar to the purple we're using? Overall it's not a big deal, just something I noticed on some of the bright or lighter posters.

Reply:
nevermind i'm stupid, hexcode